frontend/src/app/[locale]/process/ProcessNext.tsx

```diff
@@ -14,6 +14,20 @@ import {
 
 import { USWDSIcon } from "src/components/USWDSIcon";
 
+export const gitHubIssueLink = (issueNumber: number) =>
+  Object.assign(
+    (chunks: ReactNode) => (
+      <Link
+        target="_blank"
+        className="usa-link--external text-bold"
```

Comment:
the github links are bold but the other links on the page aren't. Not sure this is an issue with the branch not being rebased or not, but it looks a bit weird as is